deepchecks/core/suite.py

```diff
@@ -290,19 +290,45 @@ def to_wandb(
         with wandb_run(**wandb_kwargs) as run:
             run.log(WandbSerializer(self).serialize())
 
-    def get_failures(self) -> Dict[str, CheckFailure]:
-        """Get all the failed checks.
+    def get_checks_not_ran(self) -> List[CheckFailure]:
+        """Get all the check results which did not run (unable to run due to missing parameters, exception, etc).
 
         Returns
         -------
-        Dict[str, CheckFailure]
+        List[CheckFailure]
             All the check failures in the suite.
         """
-        failures = {}
-        for res in self.results:
-            if isinstance(res, CheckFailure):
-                failures[res.header] = res
-        return failures
+        return self.select_results(self.failures)
+
+    def get_checks_not_passed(self, fail_on_warning=True) -> List[CheckResult]:
+        """Get all the check results that have not passing condition.
```

Comment:
Not sure about this devision, where do checks without conditions go?

Maybe do this in the same devision as the new suite output: pass, fail, didnt run, other(/ checks without conditions)

Reply:
this division is just for the passed function, checks without condition are not relevant